### PR TITLE
WMS server information (including csv download format) is now taken f…

### LINKED ISSUE #1837: Metadata not included in csv downloads
App version: 4.8.4
            Build date: Tue Jul 07 16:43:25 AEST 2015
            Build number: 4
            Git ref: e1f5e0fbe9254c9fe772e7a216c50048e400d363

Steps to reproduce:

Add Argo or SOOP sst near real time collection, draw a small bounding box so not too much data requested proceed to step 3 and select csv download, once downloaded open file

What should happen:

CSV contains metatdata

What does happen:

No metadata in csv file

Same as #1235?

### PR DESCRIPTION
**Pull request by jkburges**
…rom config purely

This information is already available in the front-end - no server call is required (it was required previously when some config came from the DB).

Fixes #1837.

### CODE REVIEW

**Comment by dnahodil** [comment]
This is nice, getting rid of that redundant async call.

File: web-app/js/portal/data/LayerStore.js [hunk 1]
```diff
@@ -77,6 +51,13 @@ Portal.data.LayerStore = Ext.extend(GeoExt.data.LayerStore, {
     _serverUnrecognized: function(layerDisplayName, layerLink) {
```
Comment (dnahodil) [inline web-app/js/portal/data/LayerStore.js]: If the layerDisplayName argument is not used, should we get rid of it?

File: web-app/js/portal/data/LayerStore.js [hunk 2]
```diff
@@ -31,41 +31,15 @@ Portal.data.LayerStore = Ext.extend(GeoExt.data.LayerStore, {
...
-
-            // break this layer as the server is unrecognised
-            // openLayers needs the minimum to attempt loading #1476
-            layerLink.server = {
-                uri: this.BLOCKED,
-                type: this.BLOCKED
-            };
-            layerRecordCallback = undefined;
-            geonetworkRecord = undefined;
+            return;
```
Comment (dnahodil) [inline web-app/js/portal/data/LayerStore.js]: `return`ing here changes the behaviour from before; previously it went on to call the _addUsingLayerLinkNcwms or _addUsingLayerLinkDefault methods. If this is deliberate then can we do away with the fake server we assign to `layerLink.server`?

Comment (jkburges) [inline web-app/js/portal/data/LayerStore.js]: I'll review this - it's probably a mistake.

**Comment by jkburges** [comment]
image

**Comment by dnahodil** [comment]
I'm happy to merge when you've fixed/debunked that bug you mention.

Should we merge this now, or wait until Ext upgrade PR is merged?

**Comment by danfruehauf** [comment]
Merge before, I can rebase.

**Comment by jkburges** [comment]
GTG again @dnahodil

**Pull request merged**